torchvision/datasets/celeba.py

```diff
@@ -88,23 +92,44 @@ def __init__(
         }
         split_ = split_map[verify_str_arg(split.lower(), "split",
                                           ("train", "valid", "test", "all"))]
+        splits = self._load_csv("list_eval_partition.txt", header=None, index_col=0)
+        identity = self._load_csv("identity_CelebA.txt", header=None, index_col=0)
```

Comment:
Given that `header=None` is the default value, I would simply drop it here. If you think it is important to pass it explicitly, I would remove the default value.
```suggestion
        splits = self._load_csv("list_eval_partition.txt", index_col=0)
        identity = self._load_csv("identity_CelebA.txt", index_col=0)
```